node/src/auth/Auth.js

```diff
@@ -0,0 +1,58 @@
+const AUTH = WeDeploy.auth('auth-taste.wedeploy.io');
+
+// Data of the user signed in.
+export const currentUser = AUTH.currentUser;
+
+/**
+ * Creates a new account and signs them in on success.
+ */
+export const createAccount = (event) => {
+	event.preventDefault();
+
+	let form = event.target;
+
+	AUTH.createUser({
+		email: form.email.value,
+		name: form.name.value,
+		password: form.password.value
+	})
+		.then(() => {
+			alert('Account successfully created!');
+
+			signIn(event);
+		})
+		.catch(() => {
```

Comment:
For formatting, should we always use `{ }` for arrow functions even when we don't need them?